### ISSUE TITLE
Be more independent of ua sniffing

### ISSUE THREAD

**Issue by SirPepe** [TurbineCSS/Turbine#105]
Several plugins depend on browser sniffing for no real reason:
- Boxsizing (fixed)
- Opacity (fixed)
- Transform
- Backgroundgradient
- To some extend bugfixes

**Comment by SirPepe** [comment]
Transform plugin is fixed

**Comment by SirPepe** [comment]
Bugfix plugin is fixed

**Comment by SirPepe** [comment]
Does this work for gradients?

```
background: -moz-linear-gradient(top, #FFF, #F0F0F0) -webkit-linear-gradient(top, #FFF, #F0F0F0);
```

**Comment by SirPepe** [comment]
Fixed in 3c7326303f20d55548fb4d4afd717a53ef69247b

**Comment by SirPepe** [comment]
Fixed in 1.0.7
